airbyte-ci/connectors/pipelines/pipelines/helpers/git.py

```diff
@@ -83,3 +86,18 @@ def get_git_repo() -> git.Repo:
 def get_git_repo_path() -> str:
     """Retrieve the git repo path."""
     return get_git_repo().working_tree_dir
+
+
+@functools.cache
+def find_all_git_ignore_rules() -> List[str]:
+    """Retrieve all the git ignore rules declared in the directory."""
+    status_output = get_git_repo().git.status(["--short", "--ignored"])
+    main_logger.info("Running git status --short --ignored to find all git ignore rules")
```

Comment:
What purpose does this log message serve? Also, it's slightly wrong, "git ignore rules" are what's in `.gitignore` itself. This should read "...to find all files excluded by .gitignore" or something to that effect.

Reply:
As this operation is quite long (~20s) I wanted to add a log message to give feedback to users. I'll indeed change the message to your suggestion.